explorer/src/index.ts

```diff
@@ -41,7 +41,7 @@ router.get("/internal/kv-explorer/api/stores/:store", async (req): Promise<HttpR
 
 router.get("/internal/kv-explorer/api/stores/:store/keys/:key", async (req): Promise<HttpResponse> => {
     let store = req.params.store;
-    let key = req.params.key;
+    let key = decodeURIComponent(req.params.key);
```

Comment:
What you want it to "work" "consistently"?

Reply:
Can you rephrase, I don't quite understand what you're saying here.

Reply:
Oh, I just re-read this and I detected a whiff of sarcasm. Please confirm 😄

Reply:
My reading comprehension grows stronger by the day. Within the month I may even begin to understand irony.